<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pesto</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWebEngineView" name="webEngineView">
      <property name="locale">
       <locale language="Portuguese" country="Brazil"/>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionPrefer_ncias"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuWiki">
    <property name="title">
     <string>Navegação</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="actionMais"/>
     <addaction name="actionMenos"/>
    </widget>
    <addaction name="actionIn_cio_2"/>
    <addaction name="menuZoom"/>
    <addaction name="separator"/>
    <addaction name="actionHist_rico"/>
    <addaction name="separator"/>
    <addaction name="actionCustomizado"/>
    <addaction name="actionPesquisar"/>
    <addaction name="actionVoltar"/>
    <addaction name="actionPosterior"/>
    <addaction name="actionRecarregar"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="locale">
     <locale language="Portuguese" country="Brazil"/>
    </property>
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="actionOp_es_de_desenvolvedor"/>
   </widget>
   <widget class="QMenu" name="menuInserir">
    <property name="title">
     <string>Atalhos</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuModo">
    <property name="title">
     <string>Modo</string>
    </property>
    <addaction name="actionNavega_o"/>
    <addaction name="actionSele_o"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuWiki"/>
   <addaction name="menuInserir"/>
   <addaction name="menuModo"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionIndice">
   <property name="text">
    <string>Indice</string>
   </property>
  </action>
  <action name="actionIn_cio">
   <property name="text">
    <string>Início</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionItalico">
   <property name="text">
    <string>Italico</string>
   </property>
  </action>
  <action name="actionNegrito">
   <property name="text">
    <string>Negrito</string>
   </property>
  </action>
  <action name="actionSublinhado">
   <property name="text">
    <string>Sublinhado</string>
   </property>
  </action>
  <action name="actionC_digo">
   <property name="text">
    <string>Código</string>
   </property>
  </action>
  <action name="actionModo_sele_o">
   <property name="text">
    <string>Modo seleção</string>
   </property>
  </action>
  <action name="actionNavega_o">
   <property name="text">
    <string>Navegação</string>
   </property>
  </action>
  <action name="actionSele_o">
   <property name="text">
    <string>Seleção</string>
   </property>
  </action>
  <action name="actionC_digo_bloco">
   <property name="text">
    <string>Código (bloco)</string>
   </property>
  </action>
  <action name="actionMesmo_nivel">
   <property name="text">
    <string>Mesmo nivel</string>
   </property>
  </action>
  <action name="actionNivel_abaixo">
   <property name="text">
    <string>Nivel abaixo</string>
   </property>
  </action>
  <action name="actionNivel_acima">
   <property name="text">
    <string>Nivel acima</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="actionInterno">
   <property name="text">
    <string>Interno</string>
   </property>
  </action>
  <action name="actionInterno_2">
   <property name="text">
    <string>Interno</string>
   </property>
  </action>
  <action name="actionExterno">
   <property name="text">
    <string>Externo</string>
   </property>
  </action>
  <action name="actionParquinho">
   <property name="text">
    <string>Parquinho</string>
   </property>
  </action>
  <action name="actionPrefer_ncias">
   <property name="text">
    <string>Preferências</string>
   </property>
  </action>
  <action name="actionOp_es_de_desenvolvedor">
   <property name="text">
    <string>Opções de desenvolvedor</string>
   </property>
  </action>
  <action name="actionCustomizado">
   <property name="text">
    <string>Ir</string>
   </property>
  </action>
  <action name="actionPesquisar">
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </action>
  <action name="actionVoltar">
   <property name="text">
    <string>Anterior</string>
   </property>
  </action>
  <action name="actionPosterior">
   <property name="text">
    <string>Posterior</string>
   </property>
  </action>
  <action name="actionRecarregar">
   <property name="text">
    <string>Recarregar</string>
   </property>
  </action>
  <action name="actionMais">
   <property name="text">
    <string>Mais</string>
   </property>
  </action>
  <action name="actionMenos">
   <property name="text">
    <string>Menos</string>
   </property>
  </action>
  <action name="actionHist_rico">
   <property name="text">
    <string>Histórico</string>
   </property>
  </action>
  <action name="actionIn_cio_2">
   <property name="text">
    <string>Início</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
